registry.json

```diff
@@ -4353,17 +4353,17 @@
    "github": "nomadpool"
   },
   {
-   "ticker": "EZPZ",
```

Comment:
Your changes have removed EZPZ pool. Please only add your pool do not remove others.